.happy/terraform/modules/sfn/main.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_sfn_state_machine" "state_machine" {
               "Type": "Task",
               "Resource": "arn:aws:states:::batch:submitJob.sync",
               "Parameters": {
-                "JobDefinition": "${var.job_definition_arn}",
+                "JobDefinition": "${var.cxg_definition_arn}",
```

Comment:
not blocking, but we should either make it its own definition or rename the cxg definition to something more generic like 2_core_definition